### PR TITLE
Vuepress 2 build

### PR DESCRIPTION
**Pull request by PicoCreator**
The following is a major overhaul of the RWKV.com site to vuepress2.

You can see how it looks like here : https://rwkv-docs.tech-talk-cto.com/

The idea is to consolidate all the various links being tracked, and provide common how-to guides, or links to common project. Along with an FAQ

The primary goal is to help new users find their way around RWKV, and help reduce the support questions here on the main discord.

---

Because this FAQ will probably need to be maintained / grown, I would also recommend moving this to the RWKV group, and granting discord mods edit permission.

This also includes a code-of-conduct, taken and modified from vue.js : https://rwkv-docs.tech-talk-cto.com/community/code-of-conduct.html

---

Note that while the github build action is included, I have intentionally commented out the deploy action, as this may need to be changed on the github pages project settings first, to allow deployment to be done via actions.

### CODE REVIEW

File: docs/.vuepress/config.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+		logo: "/img/rwkv-avartar-256p.png",
+		navbar: [
+			{ text: 'Main Github', link: 'https://github.com/BlinkDL/RWKV-LM' },
+			{ text: 'Hugging Face Integration', link: 'https://huggingface.co/docs/transformers/model_doc/rwkv' },
+			{ text: 'Community Discord', link: 'https://discord.gg/bDSBUMeFpc' }
+		],
+
+		// Sidebar menu
+		sidebar: [
+			{ text: 'RWKV Lanugage Model', link: '/' },
```
Comment (guangyusong) [inline docs/.vuepress/config.ts:24]:
```suggestion
			{ text: 'RWKV Language Model', link: '/' },
```

File: docs/basic/FAQ.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+Where key information is stored in the hidden state (decided by the model weights), and is read against the current state. Effectively providing a form of trained "attention" against past tokens inside the hidden state, from the current token.
+
+## What is racoon??
+
+Racoon is a community finetuned model, for chat and general purpose use. And is made by @nath , you can find it here : https://huggingface.co/m8than/rwkv-v4-raccoon
+
+It has its own quirky personality, and is more then willing to insult you the user if needed be.
+
+## What is the RWKV chat commands?
```
Comment (guangyusong) [inline docs/basic/FAQ.md:51]:
```suggestion
## What are the RWKV chat commands?
```

File: docs/.vuepress/config.ts (hunk 1 above)
Comment (PicoCreator) [reply docs/.vuepress/config.ts]: Applied the suggestion

File: docs/basic/FAQ.md (hunk 2 above)
Comment (PicoCreator) [reply docs/basic/FAQ.md]: Applied the suggestion, thanks

**Pull request closed without merging**